### PR TITLE
Update Readme.txt

### PR DESCRIPTION
**Pull request by Theoyeah**
Sould update the psych txt

### CODE REVIEW

**Comment by HiroMizuki** [comment]
literally nobody reads this

**Comment by lsoissus** [comment]
why would you need this, nobody reads the readme.txt

**Comment by Stilic** [comment]
no

File: art/readme.txt [hunk 1]
```diff
@@ -16,11 +16,11 @@
                  /          _/           /             
 
 HEY
-Thanks for downloadin Friday Night FUNKIN'
-If you downloaded this from any other place than ninja-muffin24.itch.io/FUNKIN
+Thanks for downloadin Friday Night FUNKIN':Psych Engine
+If you downloaded this from any other place than https://gamebanana.com/mods/309789
```
Comment (HiroMizuki) [inline art/readme.txt:20]: and also, psych engine is not only available for gamebanana
my bad, this PR is definitely useless just by modifying a file that nobody reads 💀

**Comment by PersonNanoexe** [comment]
nobody reads this + its used to credit the original game lmfao

**Pull request closed without merging**